app/controllers/static_controller.rb

```diff
@@ -162,7 +162,7 @@ def favicon
 
               file&.read || ""
             rescue => e
-              AdminDashboardData.add_problem_message("dashboard.bad_favicon_url", 1800)
+              ProblemCheckTracker[:bad_favicon_url].problem!
```

Comment:
Need to add support for expiring problem check trackers.

Reply:
I am speculating now, but imagine the author didn't want to check for / reset the problem every time a favicon successfully loads, which would be the other way.